SecurityPkg/SecurityPkg.dec

```diff
@@ -230,6 +230,15 @@
   ## GUID used to generate Spdm Uid
   gEfiDeviceSecuritySpdmUidGuid = {0xe37b5665, 0x5ef9, 0x4e7e, {0xb4, 0x91, 0xd6, 0x78, 0xab, 0xff, 0xfb, 0xcb }}
 
+  ## HOB contains TPM instance which is the same as PcdTpmInstanceGuid.
+  gEdkiiTpmInstanceHobGuid           = { 0x4551b023, 0xba46, 0x4584, { 0x81, 0xcd, 0x4d, 0xe8, 0x61, 0xa7, 0x28, 0xbe } }
+
+  ## HOB contains a string of the Version of Physical Presence interface which is the same as PcdTcgPhysicalPresenceInterfaceVer.

```

Comment:
GUID used to tag the HOB indicating the Version of Physical Presence interface.
The GUIDed HOB contains the same value as PcdGetPtr (PcdTcgPhysicalPresenceInterfaceVer).

Reply:
Changed in latest code